front/src/containers/WorkflowPage/SuggestedLabels.tsx

```diff
@@ -155,7 +156,12 @@ class SuggestedLabels extends React.PureComponent<
           crowdBucketsWanted: this.props.allowedSuggestedLabels
         }}>
         {({ data, loading, error, refetch }) => {
-          if (loading || error || !data) return null;
+          
+          if (loading || error || !data){
```

Comment:
I think it would be better to display an error message if there is an error. I believe there is a standardized `Error` component in our code base so I would check to see if that is usable here first.

And I'm assuming that if there is no `data` object, it means there are no suggested labels configured but correct me if I'm off on that and it is actually something we should treat as an error. If my assumption is correct though, I would return `null`.

`if (loading) return <BeatLoader />;`
`if (error) return <Error />;`
`if (!data) return null;`

Reply:
If there a good way to force a error? Would be good to try and see what happens when it returns the Error component

Reply:
Doesn't seem like we have any sort of testing framework in place for the frontend code. So I wouldn't say it's a "good" way but for now, you could just add this above the conditionals:

`return <Error message="This is an error message" />;`

In the long run, it would probably be a good idea to have a testing framework like Jest integrated, even if we don't have a ton of tests initially. Additionally, it would be helpful to add Storybook so devs and admins can see what components are supposed to look like and if they're a good fit for a particular use case.

Reply:
If we did have a testing framework, we could use Apollo's [MockProvider](https://www.apollographql.com/docs/react/development-testing/testing/#testing-error-states) to simulate loading and error states.